packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -693,44 +693,43 @@ function wait(time) {
 }
 
 function handleErrorMsg(err) {
-  cliux.print(`Error: ${(err?.errorMessage || err?.message) ? err?.errorMessage || err?.message : messageHandler.parse('CLI_EXPORT_CSV_API_FAILED')}`, { color: 'red' })
+  cliux.print(
+    `Error: ${
+      err?.errorMessage || err?.message
+        ? err?.errorMessage || err?.message
+        : messageHandler.parse('CLI_EXPORT_CSV_API_FAILED')
+    }`,
+    { color: 'red' },
+  );
   process.exit(1);
 }
 
-async function apiRequestHandler(org, queryParam = {}) {
-  const headers = {
-    authtoken: configHandler.get('authtoken'),
-    organization_uid: org.uid,
-    'Content-Type': 'application/json',
-    api_version: 1.1,
-  };
-
-  return await new HttpClient()
-    .headers(headers)
-    .queryParams(queryParam)
-    .get(`${configHandler.get('region')?.cma}/organizations/${org?.uid}/teams`)
-    .then((res) => {
-      const { status, data } = res;
-      if (status === 200) {
-        return data;
-      } else {
-        cliux.print(`${data?.error_message || data?.message || data?.errorMessage}`, { color: 'red' });
-        process.exit(1);
-      }
+async function getAllTeamsUsingSDK(managementAPIClient, org, queryParam = {}) {
+  const teamsData = await managementAPIClient
+    .organization(org.uid)
+    .teams()
+    .fetchAll(queryParam)
+    .then((data) => {
+      return data;
```

Comment:
[MINOR] This `.then` method is doing nothing at this point of time as I do not see any logic init. We can remove it

Reply:
Removed used try and catch block